src/ubuntu/25.04/helix/Dockerfile

```diff
@@ -71,9 +72,10 @@ ENV LANG=en_US.utf8
 
 # Add MsQuic
 RUN ARCH=$TARGETARCH \
+    VERSION=$LIBMSQUIC_VERSION \
     && if [ "$TARGETARCH" = "arm" ]; then \
         ARCH="armhf"; fi \
-    && curl -LO https://packages.microsoft.com/ubuntu/24.04/prod/pool/main/libm/libmsquic/libmsquic_2.4.7_$ARCH.deb \
+    && curl -LO "https://packages.microsoft.com/ubuntu/24.04/prod/pool/main/libm/libmsquic/libmsquic_${VERSION}_$ARCH.deb" \
```

Comment:
Why not just reference `LIBMSQUIC_VERSION` directly?

Reply:
Thanks. I was having trouble with the variable resolution. I thought I had tried the correct pattern (in the follow-on commit). Apparently, I hadn't. Fixed.